packages/environment-ember-loose/__tests__/environment.test.ts

```diff
@@ -69,14 +69,6 @@ describe('Environments: Ember Loose', () => {
       '/route.js',
     ]);
 
-    // Classic component
```

Comment:
Should we keep a parts of this test or drop it all together 🤔

Reply:
We should drop everything related